crates/cosmos/cosmos-integration-tests/src/impls/bootstrap/build_cosmos_chain.rs

```diff
@@ -44,7 +44,7 @@ where
         // start up. If we don't wait, the building of the chain would fail during
         // the spawning of `ChainHandle`, as the v1 relayer tries to perform health
         // check on building.
-        bootstrap.runtime().sleep(Duration::from_secs(2)).await;
+        bootstrap.runtime().sleep(Duration::from_secs(6)).await;
 
         let chain = bootstrap
```

Comment:
I think we can now poll for `chain.query_chain_status()` here to check that the chain has started up. Previously, we couldn't do that, as the construction of `ChainHandle` itself would fail if the chain hasn't started up.

Reply:
Done in 705a8edbc45a0635d90895b27e9370e8459a4b41